x/launch/types/message_request_remove_validator_test.go

```diff
@@ -0,0 +1,58 @@
+package types_test
+
+import (
+	"testing"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/spn/testutil/sample"
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+func TestMsgRequestRemoveValidator_ValidateBasic(t *testing.T) {
+	chainID, _ := sample.ChainID(10)
+	tests := []struct {
+		name string
+		msg  types.MsgRequestRemoveValidator
+		err  error
+	}{
+		{
+			name: "invalid address",
+			msg: types.MsgRequestRemoveValidator{
+				ValidatorAddress: "invalid_address",
+			},
+			err: sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress,
+				"invalid address (invalid_address): decoding bech32 failed: invalid index of 1"),
+		}, {
+			name: "invalid chain id",
+			msg: types.MsgRequestRemoveValidator{
+				ValidatorAddress: sample.AccAddress(),
+				ChainID:          "invalid_chain",
+			},
+			err: sdkerrors.Wrapf(types.ErrInvalidChainID, "invalid_chain"),
+		}, {
+			name: "invalid chain name",
```

Comment:
I may already have commented this on another PR, what's the difference with the above?

Reply:
Well I may understand that here we use a incorrect name but good format `name-number`

In fact the test above is sufficient because we already have tests for the `ParseChainID` methods with all the different kind of invalid chain ID format.

So we just need here a test where we now the chainID fails with `ParseChainID`